db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_10_13_083210) do
+ActiveRecord::Schema[7.0].define(version: 2023_11_06_131223) do
```

Comment:
No changes in schema after the migration? 🤔

Reply:
yes wrong migration